assets/src/edit-story/components/link/index.js

```diff
@@ -23,6 +30,12 @@ export function getLinkFromElement(element) {
   return element.link || null;
 }
 
+export function inferLinkType(y) {
```

Comment:
IMHO, we're better off to just always pass the whole element here as an arg or an open object - the number of args is likely to grow. Also, pls add jsdoc.